### ISSUE TITLE
cifar_10_cnn example goes up to validation loss 2.30 in 25 epochs

### ISSUE THREAD

**Issue by MechCoder** [fchollet/keras#4587]
Hi,

The example [here ](https://github.com/fchollet/keras/blob/master/examples/cifar10_cnn.py) claims to go to 0.65 loss in 25 epochs. I tried out the code with both the theano and tensorflow backend and with the default hyper-parameters as is but it does not seem to go below 2.30 validation (and 0.1 accuracy) in 25 epochs.

Is there anything that I am doing wrong and can anyone else reproduce this? (At least a friend of mine could)

**Comment by isaacgerg** [comment]
This is for keras issues, not user help.  Please close this issue and post on the forum. Thank you!

**Comment by MechCoder** [comment]
Sorry, but the example is in the keras codebase so it definitely qualifies as an issue (since multiple people in multiple platforms are able to reproduce it). Either the example should be modified to suit the claims or the claims themselves should be removed.

**Comment by isaacgerg** [comment]
You are correct.  Sorry for my miss!

**Comment by fchollet** [comment]
Maybe the docstring is outdated. Can you copy and paste the output during training?

**Comment by MechCoder** [comment]
Thanks! It does not seem to be learning. Running the example as it is gives the following output.

This is the output from 15-25 epochs

```
Epoch 15/200
50000/50000 [==============================] - 46s - loss: 2.3035 - acc: 0.0990 - val_loss: 2.3029 - val_acc: 0.1000
Epoch 16/200
50000/50000 [==============================] - 46s - loss: 2.3034 - acc: 0.1011 - val_loss: 2.3030 - val_acc: 0.1000
Epoch 17/200
50000/50000 [==============================] - 46s - loss: 2.3034 - acc: 0.0997 - val_loss: 2.3032 - val_acc: 0.1000
Epoch 18/200
50000/50000 [==============================] - 46s - loss: 2.3033 - acc: 0.0970 - val_loss: 2.3029 - val_acc: 0.1000
Epoch 19/200
50000/50000 [==============================] - 46s - loss: 2.3034 - acc: 0.0994 - val_loss: 2.3031 - val_acc: 0.1000
Epoch 20/200
50000/50000 [==============================] - 46s - loss: 2.3033 - acc: 0.1003 - val_loss: 2.3032 - val_acc: 0.1000
Epoch 21/200
50000/50000 [==============================] - 46s - loss: 2.3033 - acc: 0.0980 - val_loss: 2.3045 - val_acc: 0.1000
Epoch 22/200
50000/50000 [==============================] - 46s - loss: 2.3033 - acc: 0.1001 - val_loss: 2.3033 - val_acc: 0.1000
Epoch 23/200
50000/50000 [==============================] - 46s - loss: 2.3034 - acc: 0.0987 - val_loss: 2.3036 - val_acc: 0.1000
Epoch 24/200
50000/50000 [==============================] - 45s - loss: 2.3034 - acc: 0.0983 - val_loss: 2.3028 - val_acc: 0.1000
Epoch 25/200
50000/50000 [==============================] - 45s - loss: 2.3033 - acc: 0.0983 - val_loss: 2.3029 - val_acc: 0.1000
Epoch 26/200
```

and from 35-50

```
Epoch 35/200
50000/50000 [==============================] - 45s - loss: 2.3033 - acc: 0.0998 - val_loss: 2.3032 - val_acc: 0.1000
Epoch 36/200
50000/50000 [==============================] - 45s - loss: 2.3034 - acc: 0.0979 - val_loss: 2.3033 - val_acc: 0.1000
Epoch 37/200
50000/50000 [==============================] - 46s - loss: 2.3032 - acc: 0.1014 - val_loss: 2.3030 - val_acc: 0.1000
Epoch 38/200
50000/50000 [==============================] - 45s - loss: 2.3032 - acc: 0.0991 - val_loss: 2.3033 - val_acc: 0.1000
Epoch 39/200
50000/50000 [==============================] - 45s - loss: 2.3033 - acc: 0.1007 - val_loss: 2.3033 - val_acc: 0.1000
Epoch 40/200
50000/50000 [==============================] - 45s - loss: 2.3033 - acc: 0.1006 - val_loss: 2.3036 - val_acc: 0.1000
Epoch 41/200
50000/50000 [==============================] - 45s - loss: 2.3034 - acc: 0.0995 - val_loss: 2.3029 - val_acc: 0.1000
Epoch 42/200
50000/50000 [==============================] - 45s - loss: 2.3033 - acc: 0.0997 - val_loss: 2.3031 - val_acc: 0.1000
Epoch 43/200
50000/50000 [==============================] - 45s - loss: 2.3034 - acc: 0.0975 - val_loss: 2.3032 - val_acc: 0.1000
Epoch 44/200
50000/50000 [==============================] - 45s - loss: 2.3033 - acc: 0.0990 - val_loss: 2.3032 - val_acc: 0.1000
Epoch 45/200
50000/50000 [==============================] - 45s - loss: 2.3033 - acc: 0.0990 - val_loss: 2.3032 - val_acc: 0.1000
Epoch 46/200
50000/50000 [==============================] - 45s - loss: 2.3033 - acc: 0.0998 - val_loss: 2.3033 - val_acc: 0.1000
Epoch 47/200
50000/50000 [==============================] - 45s - loss: 2.3034 - acc: 0.0993 - val_loss: 2.3031 - val_acc: 0.1000
Epoch 48/200
50000/50000 [==============================] - 45s - loss: 2.3034 - acc: 0.0976 - val_loss: 2.3033 - val_acc: 0.1000
Epoch 49/200
50000/50000 [==============================] - 45s - loss: 2.3032 - acc: 0.1015 - val_loss: 2.3032 - val_acc: 0.1000
Epoch 50/200
50000/50000 [==============================] - 45s - loss: 2.3033 - acc: 0.0998 - val_loss: 2.3030 - val_acc: 0.1000
```

**Comment by aidiary** [comment]
Use Adam instead of SGD #4066

**Comment by MechCoder** [comment]
Thanks. Closing as duplicate. (Would be great if you could send a PR)
